sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -128,6 +128,7 @@
           <zeebe:header key="partner1HasEmploymentInsuranceBenefits" value="$.supportingUsers.Partner1.supportingData.hasEmploymentInsuranceBenefits" />
           <zeebe:header key="partner1HasFedralProvincialPDReceipt" value="$.supportingUsers.Partner1.supportingData.hasFedralProvincialPDReceipt" />
           <zeebe:header key="partner1HasTotalIncomeAssistance" value="$.supportingUsers.Partner1.supportingData.hasTotalIncomeAssistance" />
+          <zeebe:header key="studentDataExceptionalExpense" value="$.data.exceptionalExpenseAmount" />
```

Comment:
The form.io API name must be aligned with the `studentData` variable mapped.
One of these must be changed.